src/Controller/BookController.php

```diff
@@ -85,15 +88,22 @@ public function index(Book $book, string $slug, BookRepository $bookRepository,
             }
         }
 
-        $sameAuthorBooks = $bookRepository->getWithSameAuthors($book, 6);
-
-        $myTags = $book->getTags();
-        $sameTagBooks = [];
+        $filter = null;
+        if ($book->getSerie() !== null) {
+            $filter = 'serie:!'.$book->getSerie();
+        }
 
-        if ($myTags !== null) {
-            foreach ($myTags as $tag) {
-                $sameTagBooks[$tag] = $bookRepository->findByTag($tag, 6);
-            }
```

Comment:
Should we add something to avoid the same book, like Id:!$currentId ?

Reply:
typesense does it automatically when looking for similars